core-bundle/src/Command/MigrateCommand.php

```diff
@@ -685,11 +685,13 @@ private function validateDatabaseVersion(bool $asJson): bool
             return true;
         }
 
-        $message = sprintf('Wrong database version configured, please set it to "%s"', $version);
-
-        if ($currentVersion = $this->connection->getParams()['serverVersion'] ?? null) {
-            $message .= sprintf(', currently set to "%s"', $currentVersion);
-        }
+        // If serverVersion is not configured we would actually never end up here
+        $currentVersion = $this->connection->getParams()['serverVersion'] ?? '';
```

Comment:
```suggestion
        $currentVersion = $this->connection->getParams()['serverVersion'] ?? 'NULL';
```
To make the error message more readable? But totally optional, just a nitpick ☺️

Reply:
since we never end up here, I don't think that makes sense